advertising/transaction-message/reference.md

```diff
@@ -147,14 +153,22 @@ Defines a container object that contains details about occupancy.
 |NumAdults|Required.<br />Data type is integer.<br /><br />The maximum number of adults the room is intended to accommodate.|None
 |Children|Optional.<br /><br />A container object the identifies the ages of children the room is intended to accommodate.|[Children Type](#children-type)
 
+## BedArrangement Type
+
+Defines a container object that contains details about the bed arrangement of the room.
+
+|Element|Description|Children
+|-|-|-
+|BedType|Required.<br />Data type is string.<br /><br />The type of bed in this room (for example: Extra-large double bed)|None
+|BedCount|Required.<br />Data type is integer.<br /><br />The number of bed(s) for this bed type.|None
 
 ## Children Type
 
 Defines a container object that identifies the ages of children the room is intended to accommodate.
 
 |Element|Description|Children
 |-|-|-
-|Child|Attributes:<ul><li>age&mdash;The age of the child the room is intended to accommodate.</li></ul> |None
+|Child|Attributes:<ul><li>age&mdash;The maximum age of the child the room is intended to accommodate.</li></ul> |None
```

Comment:
I still feel we should mention "each" child. Since, room can accommodate more than 1 child.